test/unittests/mongoBackend/mongoGetSubscriptions_test.cpp

```diff
@@ -96,12 +96,12 @@ static void prepareDatabaseV1Subs(void) {
                                                        "type" << "ONCHANGE" <<
                                                        "value" << BSON_ARRAY("AX2" << "AY2")
                                                        )) <<
-                        "throttling" << 5.0
+                        "throttling" << (long long) 5
                         );
 
     BSONObj sub3 = BSON("_id" << OID(SUB_OID3) <<
-                        "expiration" << 20000000 <<
-                        "lastNotification" << 25000000 <<
+                        "expiration" << (long long) 20000000 <<
```

Comment:
The problem is that we can have in existing databases expirations in the int form (i.e., without `(long long)`), so the original test is a "safety measure" to be sure that we will work for these. Maybe we should apply the "read both long/int, write always long" also to expiration?